.vscode/settings.json

```diff
@@ -1,9 +1,12 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
Can you install the prettier extension and remove this line? I think if we use the other formatter it will impact the already formatted js files.